scripts/symbolize.py

```diff
@@ -23,6 +23,10 @@
                        r'( flags .{6} (\[(?P<elf_idx>[0-9]+)\])?)?')
 ELF_LIST_RE = re.compile(r'\[(?P<idx>[0-9]+)\] (?P<uuid>[0-9a-f\-]+)'
                          r' @ (?P<load_addr>0x[0-9a-f\-]+)')
+FUNC_GRAPH_RE = re.compile(r'Function graph for TA: (?P<uuid>[0-9a-f\-]+)'
+                           r' @ (?P<load_addr>0x[0-9a-f\-]+)')
```

Comment:
You're not using `uuid` or `load_addr` parameters, are you? So better to remove them. Maybe `Function graph` is sufficient here (and would leave some flexibility if we want to tweak the message).

Reply:
Ok, will keep `Function graph` here.